added nimbus installation script

cloudnull opened the pull request.

cloudnull committed:
added nimbus installation script

Apsu commented on support-scripts/rax-nimbus-installer.sh:
```diff
@@ -0,0 +1,100 @@
+# Configure the Nimbus SNMPT
+NIMBUS_SNMPT="/opt/nimsoft/probes/gateway/snmptd/snmptd.cfg"
+if [ -f "${NIMBUS_SNMPT}" ];then
+  sed -e "63s/yes/no/;" ${NIMBUS_SNMPT} > ${WORKING_DIR}/snmptd.cfg
```
Is it safe to assume line 63 is always the right line here? Seems like it should have a line match delimiter like the previous sed

Apsu commented:
Aside from my comment above, looks good to me, though I would suggest changing the [ ] conditionals to [[ ]] for more consistent behavior, and I would also suggest using -i with sed, rather than > $dest and mv $dest $src. -e is also superfluous with a single expression to sed.

cloudnull committed:
updated script

Upon further review I have decided that this script is not in need
of setting configuration values. This is becuase the probes are
preconfigured in the package we are deploying.

Also updated [ ] to [[ ]]

cloudnull commented:
all the things were updated.
